shipyard_blueprints/snowflake/shipyard_snowflake/utils.py

```diff
@@ -109,13 +113,13 @@ def map_snowflake_to_pandas(snowflake_data_types: List[List]) -> Union[Dict, Non
             converted = snowflake_to_pandas[str(dtype).upper()]
             if converted is None:
                 raise Exception(
-                    f"Invalid datatypes: the datatype {field} is not a recognized snowflake datatype"
+                    f"Invalid datatypes: the datatype {dtype} is not a recognized snowflake datatype"
                 )
 
             pandas_dtypes[field] = converted
         except KeyError as e:
             raise Exception(
```

Comment:
should log the error message as well the error message is useful otherwise can clean remove `as e` since the variable is not being used

Reply:
Agreed, but the function itself should not log the message, but any class method that calls the function should.